app/js/components/manageApps/ManageApps.jsx

```diff
@@ -671,7 +671,9 @@ export default class ManageApps extends React.Component {
                     id="search-input"
                     ref={value => this.input = value}
                     onKeyDown={this.initiateSearch}
-                    placeholder="Need new add-ons? Type & press ENTER or clear field to display installed add-ons" />
+                    placeholder={!updatesAvailable ? 
+                      "Need new add-ons? Type & press ENTER or clear field to display installed add-ons" : 
```

Comment:
Just curious... when you say `type & press Enter`, is there a specification of what they should be typing? A name, description?